# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/Logging.h
# Commit: e178aa7d969e
# Full Hash: e178aa7d969e31550222e903b8844f80732d73bd
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-02-12 09:47:43
# Regressor Bug: 1754130
# File Overlap Count: 2
# Description:
#   Bug 1754130 - Implement SourceSurfaceWebgl for faster snapshots of DrawTargetWebgl. r=aosmond
#   
#   Now that each DrawTargetWebgl shares the same WebGL context, we can efficiently draw snapshots
#   of one DrawTargetWebgl to another without requiring any readback or error-prone driver-provided
#   shared context/resource mechanism. We just need to simply pass the WebGL texture from one target
# ==============================================================================

diff -r acbba09f0bce -r e178aa7d969e gfx/2d/Logging.h
--- a/gfx/2d/Logging.h	Fri Feb 11 19:49:55 2022 +0000
+++ b/gfx/2d/Logging.h	Fri Feb 11 19:49:55 2022 +0000
@@ -694,6 +694,9 @@
         case SurfaceType::DATA_MAPPED:
           mMessage << "SurfaceType::DATA_MAPPED";
           break;
+        case SurfaceType::WEBGL:
+          mMessage << "SurfaceType::WEBGL";
+          break;
         default:
           mMessage << "Invalid SurfaceType (" << (int)aType << ")";
           break;